libr/core/cmd_open.c

```diff
@@ -210,6 +210,7 @@ static void list_maps_visual(RIO *io, ut64 seek, ut64 len, int width, int use_co
 			for (j=0; j<width; j++) {
 				ut64 pos = min + (j*mul);
 				ut64 npos = min + ((j+1)*mul);
+				// TODO address space wraparound
 				if (s->from<npos && (s->to)>pos)
```

Comment:
missing spaces around < and >, unnecessary parenthesos around s->to

Reply:
This is just a reminder of issues that should be fixed later.